TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml

```diff
@@ -28,7 +28,7 @@
                     <th>First Name</th>
                     <th>Last Name</th>
                     <th>Born City</th>
-                    <th>Born</th>
+                    <th>Date of Birth</th>
```

Comment:
Changed the column name into more user friendly one.
https://www.w3schools.com/css/

Reply:
Good.